packages/form/src/Checkbox.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import { Label, Input } from "reactstrap";
+import classNames from "classnames";
+import { useCheckboxGroup } from "./CheckboxGroup";
+import FormGroup from "./FormGroup";
+
+const Checkbox = ({
+  label,
+  value: checkValue,
+  className,
+  ...attributes
+}) => {
+  const { value, toggle, metadata } = useCheckboxGroup(checkValue);
+
+  const classes = classNames(
+    className,
+    metadata.touched ? "is-touched" : "is-untouched",
+    metadata.touched && metadata.error && "is-invalid"
+  );
+
+  return (
+    <FormGroup for={checkValue} check inline disabled={attributes.disabled}>
+      <Input
+        name={checkValue}
+        className={classes}
+        type="checkbox"
+        {...attributes}
+        value={checkValue}
+        checked={value}
+        onChange={toggle}
+      />
+      <Label check for={checkValue}>
```

Comment:
The `for` is supposed to match the `id` of the input. Right now, clicking on the label doesn't toggle the checkbox. (same for radio)
![checkbox-label](https://user-images.githubusercontent.com/664714/59464189-c3068880-8df5-11e9-87c8-ec75ce0206a4.gif)